bot.py

```diff
@@ -99,6 +110,131 @@ async def get_last_match(ctx, name: str):
     os.remove("df_styled.png")
 
 
+@bot.command(name="add", help="Add the players to the list")
+async def add_summoner(ctx, *, message):
+    """Writes list of summoners to local
+    json file and sends the list to the bot"""
+
+    async with ctx.typing():
+        await asyncio.sleep(1)
+
+    json_path = "data/data.json"
+
+    # create a directory containing json file to store data for added summoners
+    if not os.path.exists(json_path):
+        os.makedirs("data")
+        with open(json_path, "w"):
+            pass
+
+    # accepting comma, space and comma plus space
+    players_list = [x.strip() for x in message.split(",")]
+    players_list = list(filter(None, players_list))
+
+    file_data = ""
+
+    # initializing number of players to display
+    number_of_players = 0
+
+    # initializing server id to a variable
+    server_id = str(ctx.guild.id)
+
+    # storing the json file into a variable
+    if os.path.getsize(json_path) > 0:
+        with open(json_path, "r") as file:
+            file_data = json.load(file)
+
+        # if server id exist in json, add number of players
+        if server_id in file_data:
+            number_of_players += len(file_data[server_id])
+
+    # send error message to bot then exits out of the function if an error with summoner's name
+    for count, _ in enumerate(players_list):
+        if not check_summoner_name(players_list[count]):
+            embed = discord.Embed(
+                title=":x:   Invalid Summoner Name",
+                description=f"`{players_list[count]}` is not a valid summoner name.",
+                color=discord.Color.red(),
+            )
+
+            embed.add_field(
+                name="** **",
+                value="Please enter a valid summoner name!",
+                inline=False,
+            )
+
+            embed.add_field(
+                name="** **",
+                value=f"Adding multiple summoners:\n`@{bot.user.name} add name1, name2`",
+                inline=False,
+            )
+
+            await ctx.send(embed=embed)
+            return None
+
+        # changes the incoming summoner names to what's in the api
+        players_list[count] = get_summoner_name(players_list[count])
+
+        # if summoner name in the json file, remove summoner from adding to the list
+        if os.path.getsize(json_path) > 0 and any(
+            players_list[count] in player["user_name"]
+            for player in file_data[server_id]
+        ):
+            players_list.remove(players_list[count])
+
+    # add number of players from incoming data
+    number_of_players += len(players_list)
+
+    # if more than 10 players, send error message
+    if number_of_players > 10:
+        num_of_players_needed = 10 - len(file_data[server_id])
+        await ctx.send("You have exceeded limit of 10 summoners!\n")
+        await ctx.send(f"Please add {num_of_players_needed} more summoners!\n")
```

Comment:
If case falls and this should end the call, just return here. Then you don't need another `else` on next line.

Reply:
So you don't want it to show the list of summoners added so far if it exceeds 10 people?
Just the meesage?

Reply:
sync'd on call - Print out `current list of summoners` should be ported out as separate function and called upon error.

Reply:
@jung1411 looks like issue for 1st comment has been resolved by 
```
if total_number_of_players > MAX_NUM_PLAYERS_TEAM:
            raise Exception(
                "Limit Exceeded", "You have exceeded a limit of 10 summoners!"
            )
```
But for the 3rd comment, it's not resolved yet.
When I exceed more than 10 players, I only get error message without display of `current list of summoners`
![Screen Shot 2021-07-10 at 11 17 21 PM](https://user-images.githubusercontent.com/9527124/125181653-fc787080-e1d4-11eb-9ae7-fc1622183b46.png)